docs/spec/spec.md

```diff
@@ -104,28 +132,42 @@ channel CRD can also be instantiated directly.\_
+The `ChannelSpec` defines which subscribers have expressed interest in receiving
+events from this `Channel`. It also defines the ChannelTemplate to use in order
+to create the CRD Channel backing this Channel.
 
-#### Metadata
+| Field Name        | Field Type                                    | Description                         | Constraints               |
+| ----------------- | --------------------------------------------- | ----------------------------------- | ------------------------- |
+| `channelTemplate` | [`ChannelTemplateSpec`](#ChannelTemplateSpec) | Specifies which channel CRD to use. | Immutable after creation. |
 
 #### Status
 
-| Field      | Type            | Description                                                                                  | Constraints |
-| ---------- | --------------- | -------------------------------------------------------------------------------------------- | ----------- |
-| address    | Addressable     | Address of the endpoint which meets the [_Addressable_ contract](interfaces.md#addressable). |             |
-| conditions | Conditions      | Channel conditions.                                                                          |             |
-| channel    | ObjectReference | The actual ObjectReference to the Channel CRD backing this Channel.                          |             |
+The `ChannelStatus` represents the current state of the `Channel`.
+
+| Field Name           | Field Type                                  | Requirement | Description                                                                                                            | Constraints |
+| -------------------- | ------------------------------------------- | ----------- | ---------------------------------------------------------------------------------------------------------------------- | ----------- |
+| `observedGeneration` | `int`                                       | Optional    | The 'Generation' of the `Channel` that was last processed by the controller.                                           |             |
+| `conditions`         | [`[]apis.Condition`](#apis.condition)       | Optional    | Channel conditions. The latest available observations of the resource's current state.                                 |             |
+| `annotations`        | `map[string]string`                         | Optional    | Fields to save additional state as well as convey more information to the user.                                        |             |
+| `address`            | [`duckv1.Addressable`](#duckv1.addressable) | Required    | Address of the endpoint (as an URI) for getting events delivered into the channel.                                     |             |
+| `subscribers`        | [`[]SubscriberStatus`](#subscriberstatus)   | Required    | The list of statuses for each of the channel's subscribers.                                                            |             |
```

Comment:
I think you can point to a eventingduckv1.ChannelableStatus to remove some of these fields?

Reply:
I expanded the inherited types throughout the spec and only referenced and described the ones that are used as field types. Referencing `ChannelableStatus` would break that pattern. Given this information, do you still prefer using `ChannelableStatus` here?